go/grpcwebproxy/backend.go

```diff
@@ -30,6 +30,22 @@ var (
 		"Whether to ignore TLS verification checks (cert validity, hostname). *DO NOT USE IN PRODUCTION*.",
 	)
 
+	flagBackendTlsClientCertRequired = pflag.Bool(
+		"backend_client_tls",
+		false,
+		"Whether the gRPC server of the backend requires client certificates.",
+	)
```

Comment:
I don't think we need this flag; we can unconditionally add the client cert if it's provided.

Reply:
ok ... I almost went that route originally